apps/agent-provisioning/AFJ/scripts/start_agent.sh

```
@@ -2,24 +2,32 @@
-PROTOCOL=${12}
-TENANT=${13}
-AFJ_VERSION=${14}
-INDY_LEDGER=${15}
-INBOUND_ENDPOINT=${16}
-SCHEMA_FILE_SERVER_URL=${17}
-AGENT_API_KEY=${18}
+AGENCY="$1"
+EXTERNAL_IP="$2"
+WALLET_NAME="$3"
+WALLET_PASSWORD="$4"
+RANDOM_SEED="$5"
```

Shellcheck (0.11.0)

[warning] RANDOM_SEED appears unused. Verify use (or export if used externally).

(SC2034)